packages/global/index.d.ts

```diff
@@ -73,6 +73,11 @@ declare type BlockSuiteFlags = {
    * @deprecated Will be removed after slash menu is stable
    */
   enable_append_flavor_slash: boolean;
+
+  enable_toggle_block: boolean;
+  enable_hide_children: boolean;
+  blocks_with_hidden_children: string[] | undefined;
```

Comment:
Here is the interface for feature flags only, the UI state should not be maintained here.

Also, I'm not sure if it's a good idea if the toggle state will be lost after page refresh.

Reply:
> the UI state should not be maintained here.

agreed.


> toggle state will be lost after page refresh

also agreed.

but then we need a new way to persist the toggle state for a page between refresh but not via the actual crdt model...

ideas?